### PR TITLE
Issue (No Space between add link and first link #9406 ) is Fixed.

### LINKED ISSUE #9406: [BUG] No Space between add link and first link
### Description

On the [Links page](https://www.biodrop.io/account/manage/links) page, The Edit button of the first link seems to touch the `Add Link` button. We just need a little padding between the div containing the `Add Link`, `REORDER` buttons and the first link created by the user. Refer to the screenshot for a better understanding :)

### Screenshots

image

### Do you want to work on this issue?

None

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by donishyar**
## Fixes Issue

Closes #9406 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot 2023-11-22 235514

### CODE REVIEW

**Comment by sital002** [comment]
Issue link https://github.com/EddieHubCommunity/BioDrop/issues/9406

**Comment by akash19coder** [comment]
@donishyar looks good👌

File: components/account/manage/Edit.js [hunk 1]
```diff
@@ -2,11 +2,11 @@ import PencilIcon from "@heroicons/react/24/outline/PencilIcon";
 
 export default function Edit({ href, label = "", children }) {
   return (
-    <div className="relative w-full">
+    <div className="relative w-full mt-2">
       <a
         href={href}
         aria-label={`Edit ${label}`}
-        className="rounded-full bg-secondary-medium p-2 text-white shadow-sm hover:bg-tertiary-medium focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-secondary-medium absolute top-0 left-0 pointer z-10"
+        className="rounded-full bg-secondary-medium p-2 text-white shadow-sm hover:bg-tertiary-medium focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-secondary-medium absolute top-0 -left-4 pointer z-10"
```
Comment (eddiejaoude) [inline components/account/manage/Edit.js:9]: how do these changes effect other places that use the `Edit` component?

File: components/user/UserLink.js [hunk 2]
```diff
@@ -99,7 +99,7 @@ export default function UserLink({
   );
 
   return (
-    <div className="flex flex-row gap-8 w-full">
+    <div className="flex flex-row gap-8 w-full pl-4">
```
Comment (eddiejaoude) [inline components/user/UserLink.js:102]: How does this effect the profile page?

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
This is a global change, how does this affect **all** other pages?

**Comment by donishyar** [comment]
> This is a global change, how does this affect **all** other pages?

> yes, it will affect all pages that uses page layout.

**Comment by eddiejaoude** [comment]
Yes, that is my point, did you test all other pages to make sure they still work after this change?

**Comment by donishyar** [comment]
> Yes, that is my point, did you test all other pages to make sure they still work after this change?

> Yeah , i checked it and i have'nt encounter  any issue.

**Comment by eddiejaoude** [comment]
@donishyar did you mean to close the PR?

**Pull request closed without merging**